webapps/webapp-template/src/layout/pages/TabsPage.tsx

```
@@ -108,57 +108,61 @@ export function Tabs({ tabs, activeIndex, handleTabClick }: TabToggleProps) {
-                activeIndex === index
-                  ? `2px solid ${theme.palette.customText.primary.p2.active}`
-                  : "none",
-              "&:hover": {
-                backgroundColor: "transparent",
-              },
-            }}
-          >
-            <Box
-              component="span"
-              sx={{ width: "fit-content", display: "flex", alignItems: "center" }}
+          <Box sx={{ display: "flex", flexDirection: "column", gap: 1, alignItems: "center" }}>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)